kleros-sdk/src/dataMappings/utils/lambdas.ts

```
@@ -0,0 +1,13 @@
+export const lambdas = {
+  // Converts a value to hex representation
+  hex: function () {
+    return function (text: string, render: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)